dashboard/app/models/teacher_feedback.rb

```diff
@@ -29,6 +29,7 @@ class TeacherFeedback < ApplicationRecord
   acts_as_paranoid # use deleted_at column instead of deleting rows
   validates_presence_of :student_id, :script_id, :level_id, :teacher_id, :script_level_id, unless: :deleted?
   belongs_to :student, class_name: 'User'
+
```

Comment:
If this file has no non-whitespace changes, it might be good to just revert?

Reply:
Done ✅